pkg/mcs/tso/server/server.go

```diff
@@ -60,11 +109,148 @@ func (s *Server) Close() {
 
 // GetClient returns builtin etcd client.
 func (s *Server) GetClient() *clientv3.Client {
-	return nil
+	return s.client
 }
 
 // GetHTTPClient returns builtin http client.
 func (s *Server) GetHTTPClient() *http.Client {
+	return s.httpClient
+}
+
+// AddStartCallback adds a callback in the startServer phase.
+func (s *Server) AddStartCallback(callbacks ...func()) {
+	s.startCallbacks = append(s.startCallbacks, callbacks...)
+}
+
+// GetMember returns the member.
+func (s *Server) GetMember() *member.Member {
```

Comment:
We should not return `*member.Member`.

Reply:
Yes, @lhy1024 has the plan to optimize GetMember to GetPrimary/Replica or I can do it later. I can't remove it in this pr, because @lhy1024 added this method to basic server interface and used it for resource manager. The server struct has to implement this method otherwise I need to remove it from the basic server and modify resource manager code accordingly.

Reply:
We can have a separate pr to refactor it.

Reply:
I saw @lhy1024's pr #5994 has changed to this to GetPrimary(). My pr can integrate with his pr after he merges #5994.

Reply:
ok